data_science_pipeline/peerscout/api/main.py

```diff
@@ -86,22 +76,40 @@ def create_app():
 
     app = Flask(__name__)
 
+    @app.route('/', methods=['GET'])
+    def _home():
+        html = """<h1>PeerScout Recommendation API</h1>
+        <p>This site is a prototype PeerScout Recommendation API for
+        senior and reviewing editor recommendations.</p>"""
+        return html
+
     @app.route('/api/status', methods=['GET'])
     def _status():
-        return jsonify({'message': 'Welcome to PeerScout'})
+        return jsonify({"status": "OK"})
 
     @app.route('/api/peerscout', methods=['POST'])
     @route_wrapper
     def _peerscout_api():
         data = request.get_json(force=True)
+        with open('./data_science_pipeline/peerscout/api/input-json-schema.json') as f:
```

Comment:
I would move that out of the method so that we read the file only once.

For the path you can use something like (since it is part of the package):

```python
REQUEST_JSON_SCHEMA_PATH = os.path.join(os.path.dirname(__file__), 'input-json-schema.json')
```